brave/src/test/java/brave/features/handler/CountingChildrenTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2013-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package brave.features.handler;
+
+import brave.Tracer;
+import brave.Tracing;
+import brave.handler.FinishedSpanHandler;
+import brave.handler.MutableSpan;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Test;
+import zipkin2.Span;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+
+/**
+ * This shows how a {@link FinishedSpanHandler} can add data some external formats need, such as
+ * child count.
+ *
+ * <p><em>Note:</em> this currently only works with children enclosed by their parents. If you have
```

Comment:
```suggestion
 * <p><em>Note:</em> this currently only works with children fully enclosed by their parents. If you have
```

Very small suggestion that I think makes it quite a bit more readable.